wordpress-mobile/WordPress-Android
# Quick Start for Existing Users V2 - Enable Feature Flag

Closes #16354

This PR enables the remote flag for the `Quick Start for Existing Users V2` feature.

To test:

1. Clear app data.
2. Open the app.
3. Make sure you don't manually activate the feature flag.
4. Logout/login and select "Show me around" when the quick start prompt is shown.
5. Select an existing site.
6. Notice that the new `Quick Start` card with `Get to know the WordPress app` task type is shown.

Merging Notes: 

- Wait for the https://github.com/wordpress-mobile/WordPress-Android/pull/16541 PR to be merged to trunk. ✅ 
- Wait for the https://github.com/wordpress-mobile/WordPress-Android/pull/16542 PR to be merged to trunk. ✅ 
- Wait for the connected test to pass. ✅ 
- Remove `Not Ready for Merge` label. ✅ 
- Merge the PR.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
